### DISCUSSION TITLE
[Feature] A home button in the image bar

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [immich-app/immich#4871] [Feature Request]
### The feature

When viewing photos, the normal top bar with the home button (immich logo linked to the web root) disappears and an image top bar appears. While there is a back button on the left, sometimes it takes a few clicks to get back home and other times it doesn't even do that. I am requesting the ability to have a small linked immich icon / home button somewhere in that image top bar so you can quickly get back to the home page.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
Are you thinking about the top action bar in the detail viewer?

**Reply by JesseWebDotCom** [reply]
Not sure what its called but here it is (appears after clicking on an image): 
<img width="823" alt="Screenshot 2023-11-06 at 10 39 33 AM" src="https://github.com/immich-app/immich/assets/20848952/fa5450e6-b29b-45f3-bf5c-ca7be028d13e">

**Reply by JesseWebDotCom** [reply]
Here's the back button when in a album
<img width="818" alt="Screenshot 2023-11-06 at 10 43 14 AM" src="https://github.com/immich-app/immich/assets/20848952/a2776e81-8649-4263-81a4-1e4db2355446">

so I click on a photo and to get back to the home I have to click back once to get to the album and then click again to get to the home page. I think there should be an easy way to get to the home page. also, It would be nice to always have the immich logo visible.  If this isn't for everyone, maybe this is a user preference (just like showing people in the sidebar).
